packages/tests/web-test-runner.config.js

```diff
@@ -4,218 +4,36 @@
  * SPDX-License-Identifier: BSD-3-Clause
  */
 
-import {createRequire} from 'module';
```

Comment:
I would say, let's keep the config together so that the BROWSERS env variable is usable as it was before, since this will likely change people's workflows (the tests package is useful when locally developing, not just CI)...

For example, today I'll often do
```
BROWSERS=chromium npm run test:watch
```
which will run all the tests locally just on Chrome as I'm developing. And then can change to
```
BROWSERS=preset:sauce npm run test
```
to sanity check on older browsers before pushing up a PR.

Reply:
Thanks for the explanation!

I've added that functionality back into the split config files :)

So you can go:
```
BROWSERS=chromium npm run test:watch
```
and when you need to run sauce tests, it's the same
```
BROWSERS=sauce npm run test:watch
```

If we don't wan't split config files that's fine too!

But I like how separating configs isolates a particular kind of logic. As in:
- web-test-runner.saucelabs.config.js is all about saucelabs settings
- web-test-runner.config is all about local test settings
- wtr-config.js provides shared settings that can be overwritten in either file.

When we separate these configs, a lot of the logic we do to parse details out of "browser strings" starts to look odd. The changes I've uploaded show how we can have more explicit configs without parsing strings for details.

We don't have to go 'preset:local' anymore, no more 'preset' or parsing details out of a string. To me that feels kinda awkward and although it's a neat, long term I think it's difficult to maintain or extend. A plain pojo is just as readable.

My pitch would be 'no magic' / parsed configs. WTR configs might be best served by have explicit lists of browsers and tests.

Like i said though down to change! <3

Reply:
@aomarks Seems ok... I think you set up the browser string parsing scheme?